docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -260,7 +266,7 @@ Name of the vault to access. Sub-domain of vault.azure.net.
 
 Predefined network configuration.
 Accepts a predefined network name, or file path or URL to a YAML configuration file. See the
-[Ethereum 2.0 specification] for examples.
+[consensus specification] for examples.
```

Comment:
do we just want 'consensus' rathe than 'consensus layer' here?

Reply:
The repo it links to is just "consensus specifications", so i think this one is okay.